src/decoder/mod.rs

```diff
@@ -1037,33 +1026,6 @@ impl<R: Read + Seek> Decoder<R> {
         Ok(u32::try_from(self.image().chunk_offsets.len())?)
     }
 
-    #[deprecated = "Use read_image instead"]
-    pub fn read_jpeg(&mut self) -> TiffResult<DecodingResult> {
-        self.read_image()
-    }
-
-    #[deprecated = "Use read_chunk_to_buffer instead"]
```

Comment:
Is `read_chunk` actually a good replacement yet? It has two additional arguments `chunk_index` and `output_width` to maintain.

Reply:
This version of the method requires the user to allocated the buffer themselves. Which means that they have to know the `output_width` of their buffer either way, and the extra argument just gives a bit more flexibility (though looking at `read_chunk_to_buffer` we should probably add a doc comment actually explaining what the arguments mean)